setup.py

```diff
@@ -63,74 +62,59 @@ def get_extras_require() -> Dict[str, List[str]]:
             "types-redis",
             "types-PyYAML",
         ],
-        "codecov": ["codecov", "pytest-cov"],
-        "doctest": [
+        "document": [
             "cma",
+            "lightgbm",
+            "matplotlib",
             "matplotlib>=3.0.0",
```

Comment:
duplicated

Reply:
Sorry :bow: I removed the duplication in 8251ff1.

BTW, matplotlib v3.0.0 was released in Sep 18, 2018 (3+ years ago). Can we remove the version constraint now?

Reply:
I suppose these install options are only used by Optuna developers or CI, and we can remove the lower constraint.

Reply:
👍 , removed in [461afdd](https://github.com/optuna/optuna/pull/3517/commits/461afdd4df45e3f70c1fb9de7ba0e2e6d8bb03fd).